server/src/hardware/protocol/z21/clientkernel.hpp

```diff
@@ -81,13 +84,52 @@ class ClientKernel final : public Kernel
     uint8_t m_firmwareVersionMinor;
     std::function<void(HardwareType, uint8_t, uint8_t)> m_onHardwareInfoChanged;
 
+    /*!
+     * \brief m_trackPowerOn caches command station track power state.
+     *
+     * NOTE: it must be accessed only from event loop thread or from
+     * Z21::ClientKernel::onStart().
+     *
+     * \sa EventLoop
+     */
     TriState m_trackPowerOn;
+
+    /*!
+     * \brief m_emergencyStop caches command station emergency stop state.
+     *
+     * NOTE: it must be accessed only from event loop thread or from
```

Comment:
Doxygen also supports `\note`

Reply:
Mind blowing!

Reply:
But it was part of #81 so it will be added on new commit